src/meta/contributing.md

```diff
@@ -64,10 +64,8 @@ to check.
 
 ## Style Guide
 
-Articles should be written in
-[Markdown](https://daringfireball.net/projects/markdown/syntax)
-(not [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/)).
-Markdown is usually powerful enough and is a more common technology than AsciiDoc.
+Articles should be written in Markdown—
```

Comment:
Shouldn't there be a space before the dash?

Reply:
It should be symmetric, anyway; I was hoping that ending a line with an em dash would suppress the whitespace from the newline, but it does not